modules/nodes-safe-sysctls-list.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+
+:_content-type: REFERENCE
+[id="safe_versus_unsafe_sysctls_{context}"]
+= Safe versus unsafe sysctls
+
+Sysctls are grouped into _safe_ and _unsafe_ sysctls.
+
+For system-wide sysctls to be considered safe, they must use proper namespacing and there must be proper isolation between pods on the same node. This means that if you set a sysctl for one pod it must not:
+
+- Influence any other pod on the node
+- Harm the node's health
+- Gain CPU or memory resources outside of the resource limits of a pod
+
+[NOTE]
+====
+Being namespaced alone is not sufficient for the sysctl to be considered safe.
+====
+Any sysctl not added to the allowed list on {product-title} is considered unsafe for {product-title}.
+
+All unsafe sysctls are not allowed by default. For system-wide sysctls the cluster administrator must manually enable them on a per-node basis. Pods with disabled unsafe sysctls are scheduled but do not launch.
+
+[NOTE]
+====
+You cannot manually enable interface specific unsafe sysctls.
+====
+
+{product-title} adds the following system-wide and interface specific safe sysctls to an allowed safe list:
+
+.System-wide safe sysctls
+[cols="30%,70%",options="header"]
+|===
+| sysctl | Description
+
+| `kernel.shm_rmid_forced`
+a|When set to 1, all
+shared memory objects in current ipc namespace will be automatically forced to use IPC_RMID.
+For more information see link:https://docs.kernel.org/admin-guide/sysctl/kernel.html?highlight=shm_rmid_forced#shm-rmid-forced[shm_rmid_forced].
+
+| `net.ipv4.ip_local_port_range`
+a| Defines the local port range that is used by TCP and UDP to choose the local port. The first number is the first, the second the last local port number. If possible, it is better these numbers have different parity (one even and one odd value). Must be greater than or equal to ip_unprivileged_port_start. The default values are 32768 and 60999 respectively.
```

Comment:
```suggestion
a| Defines the local port range that is used by TCP and UDP to choose the local port. The first number is the first port number, and the second number is the last local port number. If possible, it is better if these numbers have different parity (one even and one odd value). They must be greater than or equal to ip_unprivileged_port_start. The default values are 32768 and 60999 respectively.
```

Reply:
Hi @tami @jeana-redhat what is our policy on these, I simply copied these from the standard site where these definitions are officially defined.  Do we update in our docs even though that makes them then out of sync with official kernel documentation?

Reply:
Hey, sorry I missed this question. I am not sure. I am not aware of an official stance on pulling existing docs from other sources - the couple of times it has come up in my work, it has turned out that the upstream contributor was a Red Hatter and I just got his blessing to use his content. I would suggest raising this to the larger team.